Brewfile

```diff
@@ -37,6 +37,7 @@ brew "dxw/tap/whippet"
 brew "shellcheck"
 
 # JS stuff
+brew "grunt-cli"
```

Comment:
Strong disagree on this one. You should be using `npx grunt` instead or you'll end up with mismatched versions.